core-services/prow/02_config/generate-boskos.py

```diff
@@ -31,6 +31,12 @@
         'eastus2': 8,
         'westus': 8
     },
+    'azure-2-quota-slice': {
+        'centralus': 33,
+        'eastus': 8,
+        'eastus2': 8,
+        'westus': 8
+    },
```

Comment:
How were these numbers computed?

Reply:
I used the same with `azure4`.

Reply:
Which may not be correct. How do we know we have enough quota in the second account?

Reply:
Do we have a similar spreadsheet like we used for AWS2 and GCP2? If yes, have you checked the quotas? If no, can we get one?

Reply:
The 2nd subscription has exactly the same quotas as the 1st one.

Reply:
Let me correct me here. The 2nd subscription with specific quotas have been requested in https://issues.redhat.com/browse/DPP-7124. Let me /hold this and ask James for the speadsheet so we can calculate the boskos leases with an exact number.

Reply:
@wking How can we calculate this like you did in https://github.com/openshift/release/pull/12840 ?

Reply:
The commit/PR message in #12840 has the method I used previously.  Is there a particular step that wasn't clear?

Reply:
Ok so we are

- Central US:
    - 1200 standardDSv3Family / 40 = 30 clusters
    - 600 LowPriorityCores / 18 vCPUs per spot test  = 33 clusters
    - 100 Public IPs / 3 = 33 clusters
   
- eastus/eastus2/westus
  - 350 / 40 = 8 clusters
  -  350 LowPriorityCores / 18 = 19 clusters
  - 100 Public IPs / 3 = 33 clusters


So we are good to merge :)